### PR TITLE
fixing social icon color on Home page, and video paths on CardDoc

### PR DESCRIPTION
**Pull request by jwijay**
#### What does this PR do?

Fixes social icon color on Home page (Twitter icon) + video example on CardDoc (to use same video path as VideoDoc example).
#### Screenshots (if appropriate)

screen shot 2016-08-25 at 4 51 48 pm

### CODE REVIEW

File: src/Home.js [hunk 1]
```diff
@@ -114,7 +114,8 @@ export default class Home extends Component {
               <Anchor href="https://github.com/grommet/grommet"
                 icon={<GithubIcon a11yTitle='Grommet Github' />}/>
               <Anchor href="https://twitter.com/grommetux"
-                icon={<TwitterIcon a11yTitle='Grommet Twitter' />}/>
+                icon={<TwitterIcon colorIndex="grey-4"
+                  a11yTitle='Grommet Twitter' />}/>
```
Comment (alansouzati) [inline src/Home.js]: i believe there is spacing issue here

File: src/docs/components/CardDoc.js [hunk 2]
```diff
@@ -50,7 +50,7 @@ export default class CardDoc extends Component {
           description={`HPE Software Licensing and Management Solutions can
             help you optimize your software investments through control of
             complex negotiations and renewal processes`}
-          link={<Anchor href={grommetPath} 
+          link={<Anchor href={grommetPath}
             label="Learn More" icon={<LinkNextIcon />} />}
```
Comment (alansouzati) [inline src/docs/components/CardDoc.js]: spacing issue here

**Comment by alansouzati** [comment]
Thanks for your contribution @jwijay. There is only a small issue with spacing, if you can fix it that would be great.

**Comment by bryanjacquot** [comment]
Sorry if I missed it, but I don't see the documentation for the `colorIndex` property in the SocialShare documentation.

**Comment by jwijay** [comment]
@bryanjacquot I don't think the SocialShare component currently has a configurable `colorIndex` property. The grommet docs home page is using the social icon within an Anchor, and the `colorIndex` property is passed in to the Anchor `icon` prop.

Should SocialShare allow colorIndex to be passed in?

**Comment by bryanjacquot** [comment]
Thanks for the clarification, @jwijay.  @ericsoderberghp and @alansouzati, is there a reason the Grommet home page doesn't use the Social Share component?  And should SocialShare have a colorIndex property?  I thought that's what we discussed, but I could have misinterpreted.

**Comment by alansouzati** [comment]
I believe the SocialShare is intended for sharing content in a social network. The link in the home page is not to share about Grommet but to visit the Twitter page. Does it make sense?

**Pull request merged**